recipes/nctoolkit/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.1.4" %}
+
+package:
+  name: nctoolkit 
+  version: {{ version }} 
+
+  # get sha using https://api.github.com/repos/r4ecology/nctoolkit/tags
+source:
+  url: https://github.com/r4ecology/nctoolkit/archive/v{{ version }}.tar.gz 
```

Comment:
Is it possible to use the canonical pypi url:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L14

You can get the sha256 for it right on the page:

https://pypi.org/project/nctoolkit/#files

**EDIT** actually after looking at the contents of the tarball on pypi, the `requirements.txt` does not appear to be in there, which would break the source install. It's strange though since I see the file listed in the `MANIFEST.in`. You might want to check that upstream.

Reply:
Thanks. The easiest thing for me to do might be a 0.1.5 pypi release and tweak the MANIFEST.in file so that coverage.txt is included. I'll do that now and update GitHub